charts/tekton-pipelines/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.47
+version: 0.1.48
```

Comment:
Did you set up pre-commits? Our pre-commits update README by default

Reply:
@kseniyashaydurova 
fixed, had to rebase